README.md

```diff
@@ -2,11 +2,11 @@
 [![](https://vsmarketplacebadges.dev/installs-short/mlxprs.mlxprs.png)](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs)
 [![](https://vsmarketplacebadges.dev/rating-short/mlxprs.mlxprs.png)](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs)
 
-# MarkLogic Extension for Visual Studio Code
+# MLXPRS: MarkLogic Extension for Visual Studio Code
 
 _Develop, run, and debug code for MarkLogic in the popular VS Code IDE_
 
-[Visual Studio Code](https://code.visualstudio.com) is a free, cross-platform code editor and development tool from Microsoft. The free, open-source [**MarkLogic extension for VS Code**](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs) integrates MarkLogic in the cloud or on your laptop into this modern development environment.
+[Visual Studio Code](https://code.visualstudio.com) is a free, cross-platform code editor and development tool from Microsoft. The free, open-source [**MLXPRS: MarkLogic extension for VS Code**](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs) integrates MarkLogic in the cloud or on your laptop into this modern development environment.
```

Comment:
Just call it "MLXPRS" here - we've already established what it is in the title above. 

Also, the second sentence doesn't quite seem right - I think what we want to emphasize is that MLXPRS helps developers create applications. How about this instead:

"MLXPRS is a free, open-source extension for VS Code that increases the productivity of developers creating applications to unlock data agility with MarkLogic."

That might be a bit too fluffy - little less fluff here:

"MLXPRS is a free, open-source extension for VS Code that increases the productivity of developers creating applications on MarkLogic."

Also, I'm realizing that the first sentence should probably read like this:

"Visual Studio Code (link), also known as VS Code, is a free..."

That allows us to use "VS Code" in the rest of the document.